docs/Installation.md

```diff
@@ -4,12 +4,19 @@ Alternatively, it is also possible to [use AzerothCore with Docker](Install-with
 
 For macOS users: the build on mac is currently broken (see [this issue](https://github.com/azerothcore/azerothcore-wotlk/issues/122)), but you can still use AzerothCore on macOS with Docker.
 
-## 1) Installing the requirements
+## 1) Creating a user to work with on Linux
```

Comment:
Shall we mark this as an optional step? On local development environments you don't really need it

Reply:
Yeah absolutely, I’m doing this mainly because of a support question I had earlier in discord when a new Linux user wanted to do this.